Linted CSS

AllThingsSmitty opened the pull request:
I went through the sample CSS for the “Getting Started” pages,
reordered rulesets properties to an alpha order, removed properties not
needed (based on last 2 versions of major browsers and IE8+) and
removed unnecessary property values (e.g., em units on line-height).

petetnt committed:
Only update stat and clear contents when old stat is newer than current stat. Potentially fixes  #11826

AllThingsSmitty committed:
Linted CSS

I went through the sample CSS for the “Getting Started” pages,
reordered rulesets properties to an alpha order, removed properties not
needed (based on last 2 versions of major browsers and IE8+) and
removed unnecessary property values (e.g., em units on line-height).

AllThingsSmitty committed:
Linted CSS

I went through the sample CSS for the “Getting Started” pages,
reordered rulesets properties to an alpha order, removed properties not
needed (based on last 2 versions of major browsers and IE8+) and
removed unnecessary property values (e.g., em units on line-height).

abose commented:
@AllThingsSmitty  Thank you for contributing to Brackets. You will need to Accept the [Brackets CLA](http://dev.brackets.io/brackets-contributor-license-agreement.html) before we can merge this pull request.
For discussions with the brackets community, please join the [brackets slack channel](https://brackets.slack.com/) by sending a mail to admin@brackets.io with the subject line `slack registration request` specifying the email addresses you would like to register.
[brackets dev google groups link here](https://groups.google.com/forum/#!forum/brackets-dev) .

AllThingsSmitty commented:
Just signed the Brackets CLA. Thanks.

AllThingsSmitty committed:
Linted CSS

I went through the CSS under “smokes”, reordered rulesets properties to
an alpha order, removed properties not
needed (based on last 2 versions of major browsers and IE8+) and
removed unnecessary property values (e.g., em units on line-height).

AllThingsSmitty committed:
Linted CSS

I went through the CSS under “smokes”, reordered rulesets properties to
an alpha order, removed properties not
needed (based on last 2 versions of major browsers and IE8+) and
removed unnecessary property values (e.g., em units on line-height).

ficristo committed:
Rename .eslintrc to .eslintrc.json

ficristo committed:
Add gyp and gypi file extensions to python language

abose committed:
Merge pull request #12175 from petetnt/petetnt/ctrl-z-hotfix

[HOTFIX] Only update stat and clear contents when old stat is newer than current stat.

abose committed:
Merge pull request #12237 from ficristo/eslint-rename

Rename .eslintrc to .eslintrc.json

abose committed:
Merge pull request #12238 from ficristo/language-gyp

Add gyp and gypi file extensions to python language

AllThingsSmitty committed:
Merge remote-tracking branch 'origin/master'

AllThingsSmitty commented:
Closing this PR, submitted a cleaner PR with changes: https://github.com/adobe/brackets/pull/12250.
